redisinsight/api/src/modules/database-recommendation/scanner/recommendation.provider.ts

```diff
@@ -26,12 +26,10 @@ export class RecommendationProvider {
 
   private strategies: Map<string, IRecommendationStrategy> = new Map();
 
-  constructor(
-    databaseService: DatabaseService,
```

Comment:
let's remove the unused import:
`import { DatabaseService } from 'src/modules/database/database.service';`